webpack/components/extensions/HostDetails/Tabs/__tests__/errataTab.test.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import * as reactRedux from 'react-redux';
+import { renderWithRedux, patientlyWaitFor } from 'react-testing-lib-wrapper';
+import { STATUS } from 'foremanReact/constants';
+import nock, { nockInstance, assertNockRequest, mockForemanAutocomplete, mockSetting } from '../../../../../test-utils/nockWrapper';
+import { foremanApi } from '../../../../../services/api';
+import { HOST_ERRATA_KEY } from '../../HostErrata/HostErrataConstants';
+import { ErrataTab } from '../ErrataTab';
+
+const mockTraceData = require('./errata.fixtures.json');
+
+const mockHostDetails = { id: 1 };
+
+const renderOptions = { apiNamespace: HOST_ERRATA_KEY };
+const hostErrata = foremanApi.getApiUrl('/hosts/1/errata?per_page=20&page=1');
+const autocompleteUrl = '/hosts/1/errata/auto_complete_search';
+
+let firstErrata;
+let searchDelayScope;
+let autoSearchScope;
+
+beforeEach(() => {
+  const { results } = mockTraceData;
+  [firstErrata] = results;
+  searchDelayScope = mockSetting(nockInstance, 'autosearch_delay', 500);
+  autoSearchScope = mockSetting(nockInstance, 'autosearch_while_typing', true);
+});
+
+afterEach(() => {
+  nock.cleanAll();
+  assertNockRequest(searchDelayScope);
+  assertNockRequest(autoSearchScope);
+});
+
+test('Can call API for errata and show on screen on page load', async (done) => {
+  // Setup autocomplete with mockForemanAutoComplete since we aren't adding /katello
+  const autocompleteScope = mockForemanAutocomplete(nockInstance, autocompleteUrl);
+
+  // Mocking our host ID, results, and status in the test redux store
+  const useSelector = jest.spyOn(reactRedux, 'useSelector');
```

Comment:
I found that mocking useSelector caused some super-weird problems.  Instead I recommend adding to the renderOptions like I ended up doing here: https://github.com/Katello/katello/pull/9594/files#diff-0aced2c1e6b8cecf1ff1e000ed939f3ca5ac19949a73e0188301cdb3f710a02f